framework/src/base/FEProblemBase.C

```diff
@@ -4428,12 +4431,14 @@ FEProblemBase::adaptMesh()
     else
     {
       _console << "Mesh unchanged, skipping remaining steps..." << std::endl;
```

Comment:
NVM - The `flush` is flushing the statement that's put into the buffer at the top of the loop. The `endl` is only executing if the mesh doesn't change which isn't a normal execution path. It's fine the way it is.